msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md

```diff
@@ -60,6 +60,19 @@ Create the link using the following URL appended with your app ID: `https://team
 
 [Microsoft 365 Certification](/microsoft-365-app-certification/docs/certification) offers assurances that data and privacy are adequately secured and protected when a third-party Office app or add-in is installed in your Microsoft 365 ecosystem. Certification confirms that your app is compatible with Microsoft technologies, compliant with cloud app security best practices, and supported by Microsoft.
 
+## Stop app distribution
+
+You can remove apps from the [Microsoft commercial marketplace](/azure/marketplace/overview), which prevents new customers from finding and using them.
+
+**To stop distribution of an app after you've published, follow the steps:**
```

Comment:
No need to emphasize this. We discussed that this when talking about markdownlint reported warnings.